openvidu-server/deployments/enterprise/master-node/.env

```diff
@@ -258,6 +258,13 @@ OPENVIDU_STREAMS_VIDEO_MIN_SEND_BANDWIDTH=300
 # Default value is false
 # OPENVIDU_STREAMS_ALLOW_TRANSCODING=false
 
+# Request usage of Simulcast if the underlying implementation supports it.
+# The sender will generate multiple video streams with different qualities,
+# so the media server is able to select the most appropriate quality stream
+# for each Subscriber.
+# Default: true
+#OPENVIDU_STREAMS_ALLOW_SIMULCAST=true
+
```

Comment:
Not sure if this should be added only to the Enterprise `.env` or to all others too